HelpSource/Guides/UsingQuarks.schelp

```diff
@@ -250,7 +250,7 @@ list::
 ## organization
 ## country
 ## since
-## ext_dependancy - text describing external software like "PD" or "processing"
+## ext_dependency - text describing external software like "PD" or "processing"
```

Comment:
i think it'd be good to add a note that there was a typo for years

Reply:
You this so? If I make that change also in the quarks, as we briefly discussed on slack, there would be no realistic leftover of that... and this is only an info field, not used programmatically AFAIU. Web search does not reveal any quarks "in the wild" that would have this entry either.

I'm happy to add something if you still think it's desirable, but IMO we could just change it  here and the entries in relevant quark files and pretend it never happened ;)

Reply:
either solution seems reasonable to me!